README.md

```diff
@@ -94,6 +94,8 @@ Our [prioritized roadmap][roadmap] of features and utilities that the core team
 
 **FancyZones**
 - Streamlined, simplified user interface 
+- Dark mode
```

Comment:
Maybe we should specify that the for the editor.

Reply:
addressed